db/migrate/20210802074146_create_freaks.rb

```diff
@@ -5,7 +5,7 @@ def change
     create_table :freaks do |t|
       t.string :name
       t.string :description
-      t.string :photo
+      t.string :email
```

Comment:
Avoid changing old migrations. All new DB changes should happen in new migrations.

That aside, why rename this to `email`?